src/Atmos/Model/remainder.jl

```diff
@@ -62,10 +98,19 @@ function source!(rem::RemainderModel, source::Vars, state::Vars, aux::Vars, t::R
   source_s = similar(source)
   m_s = getfield(source_s, :array)
 
-  for sub in rem.subs
-    fill!(m_s, 0)
-    source!(sub, source_s, state, aux, t)
-    m .-= m_s
-  end
+  fill!(m_s, 0)
+  source!(rem.sub, source_s, state, aux, t)
+  m .-= m_s
   nothing
 end
+
+function source!(rem::RemainderModel{M,S}, source::Vars, state::Vars, aux::Vars, t::Real) where {M,S<:AtmosAcousticLinearModel}
+  source!(rem.main, source, state, aux, t)
```

Comment:
Shouldn't this have
```suggestion
 source.ρu = -state.ρ * ∇Φ
```
(because the gravitational acceleration is not taken into account in the linear model for acoustic waves)?